src/ray/gcs/gcs_server/job_info_handler_impl.cc

```diff
@@ -24,10 +25,13 @@ void DefaultJobInfoHandler::HandleAddJob(const rpc::AddJobRequest &request,
                  << ", driver pid = " << request.data().driver_pid();
   auto job_table_data = std::make_shared<JobTableData>();
   job_table_data->CopyFrom(request.data());
-  auto on_done = [job_id, request, reply, send_reply_callback](Status status) {
+  auto on_done = [job_id, request, reply, send_reply_callback](const Status &status) {
     if (!status.ok()) {
       RAY_LOG(ERROR) << "Failed to add job, job id = " << job_id
                      << ", driver pid = " << request.data().driver_pid();
+    } else {
+      RAY_LOG(DEBUG) << "Finished adding job, job id = " << job_id
```

Comment:
It is better to use `INFO` here as well as the very beginning line of this function as this RPC is very low frequency.

Reply:
fixed